static/robots.txt

```diff
@@ -0,0 +1,8 @@
+User-agent: SemrushBot
+Crawl-delay: 60
+
+User-agent: MJ12bot
+Crawl-delay: 60
+
+User-agent: bingbot
+Crawl-delay: 60
```

Comment:
Can we ask all bots to be nice, instead of just these?

Reply:
yeah, we could address it to `User-agent: *` although it's up to the bots to choose whether to obey it & I think these ones do (from experience with other sites).